documentation/sdk-manual/extensible.rst

```diff
@@ -677,6 +677,23 @@ For a recipe which inherits cmake it does:
 For a recipe which inherits meson a similar configuration is generated.
 Because there is nothing like a meson-preset a wrapper script for meson is generated.
 
+For remote debugging to work properly, the target must be running the corresponding image.
+You should also make sure that all dependencies for the recipe, as well as gdbserver
+are installed. Moreover, the target should be accessible through SSH. Lastly,
+you must make sure that the debug symbols are available in the sysroot. Here's
+an example bitbake fragment with the necessary configuration:
+
+.. code-block::
+
+   CORE_IMAGE_EXTRA_INSTALL += "gdbserver"
+
+   EXTRA_IMAGE_FEATURES += "ssh-server-openssh"
+   EXTRA_IMAGE_FEATURES += "debug-tweaks"
+
+   IMAGE_GEN_DEBUGFS = "1"
+   IMAGE_FSTYPES_DEBUGFS = "tar.bz2"
```

Comment:
IMAGE_FSTYPES_DEBUGFS is not needed. The debug symbols are taken from the $WORKDIR of the image. Generating this bz2 can be a long running task with no value.

Reply:
You are right! I tested and symbols are loaded without it. I had copied it from the doc around IMAGE_GEN_DEBUGFS. I removed this line.